### ISSUE TITLE
[HardeningKitty] ASR rules result

### ISSUE THREAD

**Issue by 0x6d69636b** [0x6d69636b/windows_hardening#10]
The ASR rules in the registry are not 100% reliable, for example when Intune is used to harden the system. The status of ASR rules should be obtained with `Get-MpPreference`.

**Comment by 0x6d69636b** [comment]
Fixed with commit 06486f5

**Comment by pmatula** [comment]
I know it's an old issue - but it's related to that topic. 

HardeningKitty detects that ASR rules are enabled (via Intune) with `Get-MpPreference`.
So you get something like this: 
```
[$] ID 10979, ASR: Block all Office applications from creating child processes (Policy), Result=0, Recommended=1, Severity=Medium
[+] ID 11029, ASR: Block all Office applications from creating child processes, Result=1, Recommended=1, Severity=Passed
```

This behavior leads to findings (and lowers your score..), right? 
It seems that Intune writes the ASR rules in: 
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Policy Manager -> ASRRules. 
The Syntax seems to be: 
`7674ba52-37eb-4a4f-a9a1-f0f9a1619a2c=1|5BEB7EFE-FD9A-4556-801D-275E5FFC04CC=1`

If HardeningKitty parses this out correctly, the "policy part" could also be evaluated? :-)
